krun/platform.py

```diff
@@ -467,10 +467,12 @@ class OpenBSDPlatform(UnixLikePlatform):
     GET_SETPERF_CMD = "sysctl hw.setperf"
 
     # flags to set OpenBSD MALLOC_OPTIONS to. We disable anything that could
-    # possibly introduce non-determinism.
-    # 's' first acts to reset free page cache to default.
+    # possibly introduce non-determinism or extra computation.
     # See malloc.conf(5) and src/lib/libc/stdlib/malloc.c for more info.
-    MALLOC_OPTS = "sfghjpru"
+    # Although 'F' would improve determinism, it seems to impact performance
+    # (enough for us to care).
+    MALLOC_OPTS = "cfgrux"
+    MALLOC_CONF = os.sep + os.path.join("etc", "malloc.conf")
```

Comment:
I think we can just say `MALLOC_CONF = "/etc/malloc.conf"` -- OpenBSD is not going to change what `os.sep` is any time soon ;)